### PR TITLE
Release 1.2.55

### PR DESCRIPTION
**Pull request by chubert-sb**
## MADiE PR

Jira Ticket: [MAT-8511](https://jira.cms.gov/browse/MAT-8511)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by chubert-sb** [comment]
Screenshot 2025-05-09 at 10 15 18 AM

File: react/components/RichTextEditor/index.jsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+                }),
+                TableRow,
+                TableHeader,
+                TableCell,
+                Underline,
+            ],
+            shouldRerenderOnTransaction: false,
+            content,
+            onUpdate: ({ editor }) => {
+                const newValue = editor.getHTML();
```
Comment (adongare) [inline react/components/RichTextEditor/index.jsx:185]: can we get this in markdown instead of HTML. markdown makes sure everything is escaped properly. here's the markdown extension- https://github.com/aguingand/tiptap-markdown

**Comment by RohitKandimalla** [comment]
I have some generic questions about this table feature 

1. if a table is added to the editor, How do we remove it ? It looks like we can do ctrl+A and delete works but not sure if users will be able to figure that out themselves.
2. If I accidentally added table twice, I don't think there is a way to remove the inner table. Should we even allow user to insert a table inside another table ?
3. How can a user add or remove columns and rows ?
4. There are no visual clues to resize the table itself or columns/rows ?
5. I am able to resize columns but not rows.
6. I know the figma only has this design a single big row with no columns and then the grid structure started from 2nd row, but are we only supporting this single structure of table ?

**Comment by RohitKandimalla** [comment]
> Yeah, I was a bit confused on what to do with the table aspect. There wasn't much direction in the ticket, so I just left it as it was in the POC and figured we'd need a follow up for it. Because to be honest there is so many options for a plethora of functionality just for that one button that needs to be worked out. I think we'll need at least another story (most likely more) to work out how we want that interaction to perform and what it should look like.
> 
> Edit: I talked with Christina and NIkki about the table and we are expected to make follow up stories for that kind of functionality since we don't fully know the details yet.

Yeah we need new stories, I also think we shouldn't be going to production with this current implementation, idk how we will be tracking it.

**Review by RohitKandimalla — APPROVED** [review]
